refgenie/asset_build_packages.py

```diff
@@ -72,6 +72,32 @@
             "cut -f 1,2 {asset_outfolder}/{genome}.fa.fai > {asset_outfolder}/{genome}.chrom.sizes",
         ]
     },
+    "fasta_txome_extract": {
+        DESC: "cDNA sequences in FASTA format exracted from the genome and annotation files, indexed FASTA (produced with samtools index) and chromosome sizes file",
```

Comment:
the description is not accurate:

- chromosome sizes file is not produced by the recipe
- FASTA index is not made available to the user